src/Management/src/TracingCore/Observer/AspNetCoreHostingObserver.cs

```diff
@@ -209,23 +193,14 @@ protected internal string ExtractSpanName(HttpContext context)
             return "http:" + context.Request.Path.Value;
         }
 
-        protected internal ISpanContext ExtractTraceContext(HttpContext context)
+        protected internal SpanContext ExtractTraceContext(HttpContext context)
         {
             var request = context.Request;
-            try
-            {
-                return Propagation.Extract(request.Headers, (d, k) =>
-                {
-                    d.TryGetValue(k, out StringValues result);
-                    return result;
-                });
-            }
-            catch (SpanContextParseException)
-            {
-                // Ignore
-            }
-
-            return null;
+            return Propagation.Extract(request.Headers, (d, k) =>
```

Comment:
Will this change of removing the try/catch result in unhandled exceptions if there are failures in here?

Reply:
No because the library caches and returns an invalid Span. Having that was giving an unreachable code warning.